### DISCUSSION TITLE
Is there a model obtained from dbnet_resnet18_fpnc_100k_synthtext.py training, which is used as a pre-training model in the paper, can you provide it?【有没有dbnet_resnet18_fpnc_100k_synthtext.py 训练得到的模型，论文中将这个模型作为预训练模型，能提供下吗？】

### DISCUSSION THREAD

**Discussion by qiyueliuhuo** [open-mmlab/mmocr#1631] [Q&A] [answered]
I didn't find this model in the project, and the original paper author's model doesn't seem to match the parameters well, can you provide a download link to this model? Thank you.【我没有在项目中找到这个模型，而原论文作者的模型似乎不能很好的匹配参数，能提供下这个模型的下载链接吗？谢谢】

**Comment by gaotongxiao** [comment, marked as answer]
It's uploaded but still in the review stage.
You can read this [page](https://github.com/open-mmlab/mmocr/blob/9691d453aabaab7293d935e8fb5d0fc654f7f7a7/configs/textdet/dbnet/README.md) for the weights and logs.

**Reply by qiyueliuhuo** [reply]
Thank you~
